### ISSUE TITLE
Do you know how to change MSA to change the predict result?

### ISSUE THREAD

**Issue by GodPCWANG** [fbarenys/alphafold_MSA#1]
I have obtained some pair MSAs through certain strategies. However, I don't know how to integrate them because the MSAs in the folder are all individual MSAs. Can you  help me?

**Comment by fbarenys** [comment]
Yes, it should follow the structure of the alignment that I attacked here.
if you want to add multiple MSAs, you should stick them next to the other, as if they were only 1 sequence multiple alignment.
[aligment_trial.txt](https://github.com/fbarenys/alphafold_MSA/files/13779954/aligment_trial.txt)

Hope it answerd your question.

**Comment by GodPCWANG** [comment]
> Yes, it should follow the structure of the alignment that I attacked here. if you want to add multiple MSAs, you should stick them next to the other, as if they were only 1 sequence multiple alignment. [aligment_trial.txt](https://github.com/fbarenys/alphafold_MSA/files/13779954/aligment_trial.txt)
> 
> Hope it answerd your question.

thank you for your answer!
